packages/kbn-esql-ast/src/types.ts

```diff
@@ -78,6 +78,17 @@ export interface ESQLLiteral extends ESQLAstBaseItem {
   value: string | number;
 }
 
+export interface ESQLStringLiteral extends ESQLLiteral {
+  type: 'literal';
+  literalType: 'string';
+  value: string;
+}
+
+export const isESQLStringLiteral = (literal: ESQLLiteral): literal is ESQLStringLiteral =>
```

Comment:
these should not live in the `types` file.
A collection of all utility method like this one is already present in `packages/kbn-esql-validation-autocomplete/src/shared/helpers.ts`

Reply:
Done in https://github.com/elastic/kibana/pull/179874/commits/9778c7db7acdb6fa76758dacab1ae62aa55eecde